src/runtime/google-analytics-event-listener-test.js

```diff
@@ -169,6 +169,51 @@ describes.realWin('GoogleAnalyticsEventListener', {}, (env) => {
     await eventManager.lastAction_;
   });
 
+  it('Should log ga and gtag event with additional gaParams and gtagParams', async () => {
+    setupEnvironment(
+      Object.assign({}, env.win, {
+        ga: () => {},
+        gtag: () => {},
+      }),
+      true
+    );
+    const gaEvent = analyticsEventToGoogleAnalyticsEvent(
+      AnalyticsEvent.IMPRESSION_OFFERS
+    );
+    const gaEventWithParams = Object.assign({}, gaEvent, {
+      eventCategory: 'TEST CATEGORY',
```

Comment:
Can we set an empty value (or a different value) for `eventCategory` to make sure it's overwritten by the given params?

Reply:
Never mind. It's actually overwriting what's returned by `analyticsEventToGoogleAnalyticsEvent`. I misunderstood it originally.